src/unix/sunos.c

```diff
@@ -865,3 +865,13 @@ void uv_free_interface_addresses(uv_interface_address_t* addresses,
 
   uv__free(addresses);
 }
+
+#if !defined(_POSIX_VERSION) || _POSIX_VERSION < 200809L
```

Comment:
arent we missing the defined(__sun) in here?

Reply:
A __sunos define in sunos.c shouldn't be required, right? :)

Reply:
My bad. Did not pay attention to the file name 😔. Really sorry.